moped-editor/package.json

```diff
@@ -41,6 +41,7 @@
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@material-ui/styles": "^4.10.0",
     "apollo-boost": "^0.4.9",
+    "atd-kickstand": "github:cityofaustin/atd-kickstand#4616-sg-abastract",
```

Comment:
This works for now, but we'll want to figure out some semver pattern with kickstand I think and use a non-feature branch. I bet you've already thought about this too.

Reply:
Also just realized, if we delete this branch from Github (like we normally do after merging a feature branch) it will break.

Reply:
Let me make it the main branch

Reply:
After updating it to main branch, it looks like it works on my end:

![2020-12-04_15-06-36](https://user-images.githubusercontent.com/5282430/101215069-bf4a6f80-3642-11eb-8e9c-b71fffc3ecbd.png)